scripts/rouletteSender.js

```diff
@@ -0,0 +1,41 @@
+const PluginBtp = require('ilp-plugin-btp')
+const ILP = require('..')
+
+;(async function test () {
+  const plugin1 = new PluginBtp({ server: 'btp+ws://:plugin1@localhost:8912/' })
+  const plugin2 = new PluginBtp({ server: 'btp+ws://:plugin2@localhost:8912/' })
+  await plugin1.connect()
+  await plugin2.connect()
+  const loop = await ILP.LT.createLoop(plugin1, plugin2)
```

Comment:
Does this send plugin1 to plugin2? It might be clearer if we made this an object with names for the two plugins (i.e. outgoing vs. incoming, or sender vs. receiver), rather than using positional arguments.